frontend/public/components/_dropdown.scss

```diff
@@ -164,6 +164,22 @@ $color-bookmarker: #DDD;
       white-space: nowrap;
     }
   }
+
+  .co-create-from-yaml {
+    margin-left: auto;
+
+    &:hover {
+      background-color: transparent !important;
```

Comment:
Is this needed if we don't use the `co-m-nav-link` style? It might be better to have our own `co-m-masthead-link` or similar